Lib/concurrent/futures/interpreter.py

```diff
@@ -0,0 +1,188 @@
+"""Implements InterpreterPoolExecutor."""
+
+import pickle
+from . import thread as _thread
+import _interpreters
+import _interpqueues
+
+
+LINESEP = '''
+'''
```

Comment:
Why not just make this `\n`?
```suggestion
LINESEP = '\n'
```

Reply:
I've dropped it.